.github/workflows/ci.yml

```diff
@@ -0,0 +1,59 @@
+name: CI
+
+on:
+  push: # Run on pushes to "develop"
+    branches: 
+      - develop
+  pull_request: # Run on PRs to "develop"
+    branches: 
+      - develop
+    types: [opened, reopened, synchronize, ready_for_review]
+
+jobs:
+  prettier:
+    if: ${{ github.event.pull_request.draft == false }} 
+
+    name: Check Javascript code style
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
In order to avoid repeating repo cloning and packages installation why not share the same job?
Another solution is to use `actions/cache`. What do you think?

Reply:
Separating the actions into two jobs allows us to more easily understand what exactly is failing. Caching is also already done with `cache: npm` and it seems that they are indeed sharing the same cache as per what I'm able to see on the Actions tab.

Reply:
Oh, I didn't see that